src/bin/options/share-envs.ts

```diff
@@ -0,0 +1,7 @@
+import Mocha from '../../main/mocha';
+
+export default function applyShareEnvs(mocha: Mocha, shareEnvs?: boolean) {
+  if (shareEnvs) {
+    mocha.setEnvs(require('worker_threads').SHARE_ENV);
```

Comment:
Note that we do the `require` only in the case that the user asked for this action.
We do so because `worker_threads` does not always exists

Reply:
IIRC `worker_threads` doesn't exist for older Node.JS versions. So what will happen if the user specifies `--share-env` in an older Node.JS version? Do you think it's worth it to exit(1) with an error saying this is not supported?

Reply:
I'll do it 👍

Reply:
I'll do it 👍